src/null-safety/index.md

```diff
@@ -78,8 +78,8 @@ Dart null safety support is based on the following three core design principles:
 
 ## Dart 3 and null safety
 
-Dart 3---planned for a mid-2023 release---always has sound null safety.
-Dart 3 will prevent code from running without it.
+Dart 3 always has sound null safety,
+and will prevent code from running without it.
 
```

Comment:
I would link to the full migration guide right at the top of this section (maybe you did somewhere else on the page but I didn't see it). I know the plan is to dismantle these pages eventually but for now, someone could end up here and want more info.

```suggestion
For guidance on migrating to Dart 3, check out the [Dart 3 migration guide][/resources/dart-3-migration]
```